navigation/presentation.html

```diff
@@ -2,43 +2,1222 @@
+              <br /><br />
+              There are also cons of using FAB.
+              <br /><br />
+              - A floating action button can distract users from content.
+              <br /><br />
+              - It can block content.
+              <br /><br />
+              - It is also icon-only navigation. The problem is that icons are sometimes hard to understand because they’re so open to interpretation.
+            </aside>
+          </section>
+          <!-- 3.5 slide -->
+          <section class="bullets-and-picture-slide">
+            <h4 class="slide-name">gesture-based navigation</h4>
+            <div>
+              <img src="assets/images/gesture-base_navigation.png" alt="gesture-base navigation" />
+              <div class="pros-cons-section">
+                <p class="big">
+                  Pros:
+                </p>
+                <ul>
+                  <li>
+                    make interfaces more minimal and save screen space for valuable content
+                  </li>
+                  <li>
+                    UI is more natural
+                  </li>
+                </ul>
+                <p class="big">
+                  Cons:
+                </p>
+                <ul>
+                  <li>
+                    navigation is invisible
+                  </li>
+                  <li>
+                    user effort increases
+                  </li>
+                </ul>
+              </div>
+            </div>
+
+            <aside class="notes">
+              Gesture-based navigation enables users to quickly swipe in their desired direction to navigate through an app or perform a particular action.
+              <br />
+              Using gestural navigation helps to create a journey between different scenes within an app and gestures include touching and dragging, both horizontally and vertically, as well as zooming in and out. It’s highly immersive and interactive, creating a dynamic experience.
+              <br /><br />
+              The great thing about this pattern is that it’s relatively simple to grasp for even the most inexperienced user as the gestures are often intuitive and only require a little experimentation to get right.
+              <br /><br />
+              But if the navigation is cluttered, then without proper visual hints and cues, users can become confused about how to interact with the app.
+              <br />
+              This will increase the effort of the user.
+            </aside>
+          </section>
+          <!-- 3.6 slide -->
+          <section class="bullets-and-picture-slide">
+            <h4 class="slide-name">3D touch</h4>
+            <div>
+              <img src="assets/images/3d_touch.jpg" alt="3d touch" />
+              <div class="pros-cons-section">
+                <p class="big">
+                  Pros:
+                </p>
```

Comment:
![Screenshot 2023-02-16 at 00 25 19](https://user-images.githubusercontent.com/1831217/219147320-4c368b18-17ac-4105-ba1e-63c1adf868c8.png)
words are too close to the picture (on FullHD resolution)

Reply:
fixed